etc/cumulus-config/cumulus-config.yml

```diff
@@ -84,6 +85,24 @@ cumulus_iris_euclid_users:
     password: "{{ federated_user_default_password }}"
     roles: "{{ cumulus_user_roles }}"
 
+cumulus_iris_gaia:
+  name: iris-gaia
+  description: IRIS@Cambridge Gaia
+  project_domain: Default
+  users: "{{ cumulus_iris_gaia_users }}"
+  user_domain: "{{ federated_domain }}"
+  quotas: "{{ iris_small_quota }}"
+
+cumulus_iris_gaia_users:
+  - name: "Nigel Hambly"
+    email: "nch@roe.ac.uk"
```

Comment:
So... we need these users to be in the system already.

Might be eaiser just you add yourself for now, so we can just create the project, then add the users once they appear.